gomint-api/pom.xml

```diff
@@ -99,6 +99,9 @@
                             <head>API stability level:</head>
                         </tag>
                     </tags>
+                    <sourceFileExcludes>
+                        <sourceFileExclude>module-info.java</sourceFileExclude>
```

Comment:
Whats the reason behind this exclude? Just so that it doesn't show up in Docs?

Reply:
The compile process failed without the exclusion (for me)

Reply:
Instead, the javadoc plugin should be further specified by adding a version. Javadoc generration does not fail on my jenkins and I do only inject javadoc plugin version 3.2.0 (and quiet to true and --frames option)